dnsmasq/files/dnsmasq.init

```diff
@@ -455,6 +460,8 @@ dhcp_add() {
 		DNS_SERVERS="$DNS_SERVERS $dnsserver"
 	}
 
+	dnsmasq_device_list="${dnsmasq_device_list} $ifname"
+	dnsmasq_interface_list="${dnsmasq_interface_list} $net"
```

Comment:
This is a list variable that call itself and this is used later at line 1115 and 1124